packages/core/src/renderer.ts

```diff
@@ -315,7 +321,18 @@ export interface ExitActionResolver {
 }
 
 function replaceInlineContext(target: string, context: any): string {
-  return target.replace(/{{([^{{}}]+)}}/g, (_, key) => get(context, key, ``));
+  const output = Mustache.render(target, {
+    ...context,
+    decimals: function () {
```

Comment:
with this pattern, this list of functions is going to grow very long and it'll be bundled into every app.